README.md

```diff
@@ -171,6 +173,14 @@ This variation has been modified from the Row Stagger base layout by rotating th
 
 ---
 
+## Etymology
+
+"Canary" comes from the phrase "canary in a coal mine". Adopters of this layout are likened to miners, and the mine is a layout with extremely high roll stats.
```

Comment:
This feel incomplete. The coal mine metaphor is accurate, but when I asked, the answer I got was this was akin to a "canary deploy", a trial run to test an unproven idea. The unproven idea being focusing on high rolls. The metaphor extending to the mine and miners feels a little stretched :)

Reply:
While that is the more modern usage of the term, in the context of a "canary deploy", it is serving the same purpose as the mine metaphor; as a canary deploy is generally a microcosm or incremental step to test the waters, see if the changes being deployed work as expected. So I think either is apt here.